### PR TITLE
Adapt README in charts (I)

### PR DESCRIPTION
**Pull request by carrodher**
```
.
├── ## Parameters
│  └── (Table)
├── ## Configuration and installation details
│  ├── ### Rolling VS Immutable tags
│  ├── ### Production configuration and horizontal scaling
│  └── ### Forwarding the logs to another service
└── ## Persistence
```
Batch I: From Airflow to Harbor

### CODE REVIEW

File: bitnami/cassandra/README.md [hunk 1]
```diff
@@ -225,31 +207,41 @@ You can enable TLS between client and server and between nodes. In order to do s
...
+The [Bitnami cassandra](https://github.com/bitnami/bitnami-docker-cassandra) image allows having initialization scripts mounted in `/docker-entrypoint.initdb`. This is done in the chart by adding files in the `files/docker-entrypoint-initdb.d` folder (in order to do so, clone this chart) or by setting the `initDBConfigMap` value with a `ConfigMap` (named, for example, `init-db`) that includes the necessary `sh` or `cql` scripts:
 
 ```console
-helm install --name my-release bitnami/cassandra --set cluster.internodeEncryption=all \
-             --set cluster.clientEncryption=true --set tlsEncryptionSecretName=cassandra-tls \
+initDBConfigMap=init-db
 ```
 
-## Initializing the database
+### Using a custom Cassandra image
```
Comment (beltran-rubo) [inline bitnami/cassandra/README.md]: I do not think we are adding valuable information for the user, we can remove this section though

File: bitnami/elasticsearch/README.md [hunk 2]
```diff
@@ -391,21 +398,6 @@ As an alternative, this chart supports using an initContainer to change the owne
 
 You can enable this initContainer by setting `volumePermissions.enabled` to `true`.
 
-## Troubleshooting
```
Comment (beltran-rubo) [inline bitnami/elasticsearch/README.md]: this section is useful for users no? that is a very common error

Comment (carrodher) [reply bitnami/elasticsearch/README.md]: Yes, I moved it inside the Configuration section, take a look some lines above

File: bitnami/cassandra/README.md (hunk 1 above)
Comment (carrodher) [reply bitnami/cassandra/README.md]: Makes sense, I just removed it

**Pull request merged**